pkgs/by-name/fi/fish/package.nix

```diff
@@ -350,6 +352,7 @@ stdenv.mkDerivation (finalAttrs: {
   checkTarget = "fish_run_tests";
   preCheck = ''
     export TERMINFO="${ncurses}/share/terminfo"
+    export PATH=${testPython}/bin:$PATH
```

Comment:
This should be able to be dropped if you explicitly substitute all PATH lookups with absolute paths.